<launch>
    <node pkg="ssafy_2" type="mgeo_pub.py" name="mgeo_pub"  />
    <node pkg="ssafy_2" type="gpsimu_parser.py" name="gpsimu_parser" />
    <node pkg="ssafy_2" type="global_path_pub.py" name="global_path_pub"  />
    <node pkg="ssafy_2" type="local_path_pub.py" name="local_path_pub" />    
    <node pkg="ssafy_2" type="tf_pub.py" name="tf"  />
    <node pkg="ssafy_3" type="image_lane_fitting.py" name="image_lane_fitting" />
    <node pkg="ssafy_3" type="gps_db_connection.py" name="gps_db_connect" />
    <node pkg="ssafy_3" type="check_path.py" name="check_path" />
    <node pkg="ssafy_3" type="lattice_planner.py" name="lattice_planner" output="screen" args="/Object_topic"/>
    <!-- <node pkg="ssafy_3" type="advanced_purepursuit.py" name="advanced_purepursuit"  args="/selected_path"/> -->

    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find ssafy_3)/rviz/lane_detection_rviz.rviz" />
</launch>